### PR TITLE
fix: Simplify wording to make more sense

### PR DESCRIPTION
**Pull request by emerconn**
I think that "count odd numbers backward by twos" instead of "count backward by twos by odd numbers" is easier to understand.

Another option which sounds nice: "count backward by twos using odd numbers"

---
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

### CODE REVIEW

**Review by RandellDawson — CHANGES_REQUESTED** [review]
Small suggestion.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/count-backwards-with-a-for-loop.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ for (var i = 10; i > 0; i -= 2) {
 }
 ```
 
-`ourArray` will now contain `[10,8,6,4,2]`. Let's change our initialization and final expression so we can count backward by twos by odd numbers.
+`ourArray` will now contain `[10,8,6,4,2]`. Let's change our initialization and final expression so we can count odd numbers backward by twos.
```
[the remark this answers is not in the document]
Comment (emerconghaile) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/count-backwards-with-a-for-loop.md:25]: I think this sounds even better :)

[the remark this answers is not in the document]
Comment (RandellDawson) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/count-backwards-with-a-for-loop.md:25]: I you like it, click the Commit suggestion button above and provide an applicable commit message.

**Review by RandellDawson — APPROVED** [review]

**Review by naomi-lgbt — APPROVED** [review]
Looks good to me.

**Pull request merged**